src/modules/drive/AddMenu/AddMenuContent.jsx

```
@@ -109,7 +109,7 @@ const AddMenuContent = forwardRef(
             </>
           )}
         {!isEncryptedFolder &&
-          !isFolderFromSharedDriveRecipient(displayedFolder) && (
+          !isFromSharedDriveRecipient(displayedFolder) && (
```

ESLint

[error] Delete `⏎·········`

(prettier/prettier)